src/components/UnpublishedListing/UnpublishedListingCardMeta.css

```diff
@@ -0,0 +1,23 @@
+.cardMeta {
```

Comment:
Import `../UI/theme` and use variables where it makes sense, especially for colors (add new ones where necessary). Doesn't have to be over-thought - lots of change is coming. Just want to establish the pattern of general re-use for these kinds of values.